libraries/botbuilder/tests/cloudAdapter.test.js

```diff
@@ -150,7 +150,8 @@ describe('CloudAdapter', function () {
             await adapter.process(req, res, logic);
 
             assert.equal(StatusCodes.UNAUTHORIZED, res.statusCode);
-            expect(consoleSpy.calledWithMatch({ message: 'jwt expired' })).to.be.true;
+            expect(consoleStub.calledWithMatch({ message: 'The token has expired' })).to.be.true;
+            consoleStub.restore();
```

Comment:
If we use the sandbox, the restore is not necessary because it does a restore of the sandbox after each test (line 36).